docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+
+      * - Astrophysics
+        - `Cholla <https://github.com/amd/InfinityHub-CI/tree/main/cholla/>`_
+        - Static-mesh, GPU-native hydrodynamics simulation code that efficiently
+          runs high-resolution simulations on massively-parallel computers.
+          Cholla is written in a combination of C++, :doc:`HIP <hip:index>`, and
+          CUDA C, and requires at least one AMD accelerator to run.
+
+      * - Geophysics
+        - `SPECFEM3D Cartesian <https://github.com/amd/InfinityHub-CI/tree/main/specfem3d>`_
+        - Simulates acoustic (fluid), elastic (solid), coupled acoustic/elastic,
+          poroelastic or seismic wave propagation in any type of conforming mesh
+          of hexahedra (structured or otherwise). It can, for instance, model
+          seismic waves propagating in sedimentary basins or any other regional
+          geological model following earthquakes. It is also used for
+          non-destructive testing and for ocean acoustics.
+
+      * - Molecular dynamics
+        - `GROMACS with HIP (AMD implementation) <https://github.com/amd/InfinityHub-CI/tree/main/gromacs>`_
+        - Recipe based on an
+          `AMD fork of the GROMACS project <https://github.com/ROCm/gromacs>`_
+          optimized for AMD accelerators. GROMACS is a versatile package to
+          perform molecular dynamics simulations of systems with hundreds to
+          millions of particles.
+
+      * -
+        - `LAMMPS <https://github.com/amd/InfinityHub-CI/tree/main/lammps>`_
+        - Large-scale Atomic/Molecular Massively Parallel Simulators: a
+          classical molecular dynamics library with a focus on materials
+          modeling. It is capable of modeling 2D and 3D systems composed of a
+          few up to billions of particles using AMD Instinct accelerators via
+          the Kokkos backend.
+
+      * - Computational fluid dynamics
+        - `NEKO <https://github.com/amd/InfinityHub-CI/tree/main/neko>`_
+        - Portable framework written in modern Fortran for high-order spectral
+          element flow simulations. Using an object-oriented design, Neko allows
+          for multi-tier abstraction for solver stacks and allows Neko to be
+          built against various types of hardware backends.
+
+      * -
+        - `nekRS <https://github.com/amd/InfinityHub-CI/tree/main/nekrs>`_
+        - Open-source Navier-Stokes solver based on the spectral element method
+          targeting CPUs and accelerators that support :doc:`HIP <hip:index>`,
+          CUDA, or OpenCL. 
+
+      * - Computational chemistry
+        - `QUDA <https://github.com/amd/InfinityHub-CI/tree/main/quda>`_
+        - Library designed for efficient lattice QCD computations on
+          accelerators. It includes optimized Dirac operators and a variety of
+          fermion solvers and conjugate gradient (CG) implementations, enhancing
+          performance and accuracy in lattice QCD simulations.
```

Comment:
```suggestion
        - Library designed to efficiently lattice QCD computations on
          accelerators. It includes optimized Dirac operators and a variety of
          fermion solvers and conjugate gradient (CG) implementations, enhancing
          performance and accuracy in lattice QCD simulations.
```

Reply:
"Lattice" is not a verb here. It's modifying QCD.